docs/_pages/installation.md

```diff
@@ -135,6 +140,10 @@ It contains Gazebo world files and configuration files required for the Academy
     sudo apt install ros-melodic-jderobot-assets
     ```
 
+3. Source the environment variables:
+    ```bash 
+    source ~/.bashrc    
+    ```
 
```

Comment:
```bash
    echo "source /opt/jderobot/share/jderobot/gazebo/assets-setup.sh" >> ~/.bashrc
    source ~/.bashrc
    ```

Reply:
adding this one may help but it is not neccassry for all the exercises
source /opt/jderobot/share/jderobot/gazebo/gazebo-setup.sh